addons/web/static/src/views/calendar/calendar_renderer.scss

```diff
@@ -170,56 +256,56 @@
             // Row containing "all day" events
             div.fc-day-grid {
                 background-color: $o-view-background-color;
-                box-shadow: 0 6px 12px -6px rgba(black, 0.16);
-                border-left-color: rgba($o-brand-odoo, 0.3);
-                border-right-color: rgba($o-brand-odoo, 0.3);
+                box-shadow: 0 6px 12px -6px rgba($gray-900, 0.16);
```

Comment:
`$gray-900` is almost white in dark-mode 😉
![image](https://github.com/odoo/odoo/assets/8558351/ffd5ac4a-5ace-4e99-ba2c-f1ed9a006a9b)

Shadows must be always black